experiments/benchmarks/benchmark_madgraph/exec_benchmark.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+
+#This is the main execution script. It installs MG5 and LHAPDF if not present, executes MadGraph5_aMC, 
+#generates the matrix elements and then performs MC integration for all subprocesses.
+
+
+full_path=$(realpath $0)
+dir_path=$(dirname $full_path)
+./install_mg.sh
+export LD_LIBRARY_PATH=$dir_path/MG5_aMC_v2_8_2/HEPTools/lhapdf6_py3/lib
+if [ -d "fortran_output" ]; then rm -Rf ./fortran_output; fi
+python3 MG5_aMC_v2_8_2/bin/mg5_aMC --logging=ERROR --file=nis_script_dd.mg5 2>&1> mg5_log.txt 2>&1
+cd ./fortran_output/SubProcesses
+for f in *; 
+    do   if [ -d "$f" ]; 
+        then cd $f; 
+        make matrix2py.so 2>&1> f2py_log.txt 2>&1;
+        rm -f matrix2py.so
+        mv matrix2py* matrix2py.so
+        mkdir -p ../../../../utils/integrands/mg/$f/param 
+        cp ../../Cards/param_card.dat ../../../../utils/integrands/mg/$f/param 
+        cp {nexternal.inc,pmass.inc,param.log} ../../../../utils/integrands/mg/$f/param 
+        cp matrix2py.so ../../../../utils/integrands/mg/$f
+        python3 ../../../../benchmark_madgraph.py --process $f --pdf_type NNPDF23_nlo_as_0119   --pdf_dir ../../../MG5_aMC_v2_8_2/HEPTools/lhapdf6_py3/share/LHAPDF --db madgraph_$f --experiment_name madgraph_$f --config ../../../../benchmark_config_examples/madgraph_grid_config.yaml --debug --lhapdf_dir ../../../MG5_aMC_v2_8_2/HEPTools/lhapdf6_py3/lib/python3.*/site-packages --pdf
+        mv madgraph_$f $dir_path/
```

Comment:
Could you please assign names to the folders you refer to with these ../../ etc (I guess there is the benchmark folder and the benchmark_madgraph folder)? It would make things more readable.